torch_object_detection_segmenter.py

```diff
@@ -186,9 +186,13 @@ def _crop_image(img, target_size: Union[Tuple[int, int], int], top: int = None,
                     logging.debug(
                         f'detected {label_name} with confidence {score} at position {(top, left)} and size {target_size}')
                     batched.append(
-                        dict(offset=0, weight=1., blob=_img,
-                             location=(top, left), tags={'label': label_name}))
+                        Document(dict(offset=0, weight=1., blob=_img,
```

Comment:
do not use batches, do directly doc.chunks.append()

Reply:
Done